php/php-zstd/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           php 1.1
+
+github.setup        kjdev php-ext-zstd 0.7.4
+fetch.type          git
+
+name                php-zstd
+categories-append   devel
+platforms           darwin
+maintainers         nomaintainer
+license             Apache-2
+
+homepage            https://github.com/kjdev/php-ext-zstd
```

Comment:
The homepage line can be removed because the GitHub portgroup already sets it to this value by default.

Reply:
Thanks @mf2k ,

I've removed the homepage and, also, changed the license to MIT (following the LICENSE in the repo).